src/cursor/cur_std.c

```diff
@@ -1019,6 +1019,29 @@ __cursor_config_debug(WT_CURSOR *cursor, const char *cfg[])
     return (0);
 }
 
+/*
+ * __check_prefix_format --
+ *     Check if the schema format is a fixed-length string, variable string or byte array.
+ */
+static int
+__check_prefix_format(const char *format)
+{
+    size_t len;
+    const char *p;
+
+    /* Early exit if prefix format is S or u. */
+    if (WT_STREQ(format, "S") || WT_STREQ(format, "u"))
+        return (0);
+    /*
+     * Now check for fixed-length string format through looking at the characters before the NULL
+     * character.
+     */
```

Comment:
`NULL` is the pointer, `nul` is the name of the byte.

Reply:
thanks! I was wondering what was the difference between the two.